### ISSUE TITLE
Repo manager move mode cross-device error

### ISSUE THREAD

**Issue by david4096** [ga4gh/server#1127]
I received this error when trying to move a file from one drive to another. GA4GH is installed on /srv, while the file is in /run/shm.

```
(newserverenv)ga4gh-admin@ga4gh-demo:/srv/ga4gh$ ga4gh_repo add-featureset ga4gh-example-data demo /run/shm/1000Genomes/gencode_v24lift37.db --moveMode=move
Traceback (most recent call last):
  File "/srv/ga4gh/server/newserverenv/bin/ga4gh_repo", line 9, in <module>
    load_entry_point('ga4gh==0.2.3.dev4+nge0b07f3', 'console_scripts', 'ga4gh_repo')()
  File "/srv/ga4gh/server/newserverenv/local/lib/python2.7/site-packages/ga4gh-0.2.3.dev4+nge0b07f3-py2.7.egg/ga4gh/cli.py", line 1953, in repo_main
    runner.run()
  File "/srv/ga4gh/server/newserverenv/local/lib/python2.7/site-packages/ga4gh-0.2.3.dev4+nge0b07f3-py2.7.egg/ga4gh/cli.py", line 1728, in run
    self.datasetName, self.filePath, self.moveMode)
  File "/srv/ga4gh/server/newserverenv/local/lib/python2.7/site-packages/ga4gh-0.2.3.dev4+nge0b07f3-py2.7.egg/ga4gh/repo_manager.py", line 526, in addFeatureSet
    self._moveFile(filePath, destPath, moveMode)
  File "/srv/ga4gh/server/newserverenv/local/lib/python2.7/site-packages/ga4gh-0.2.3.dev4+nge0b07f3-py2.7.egg/ga4gh/repo_manager.py", line 200, in _moveFile
    os.rename(src, dst)
OSError: [Errno 18] Invalid cross-device link
```

**Comment by jeromekelleher** [comment]
Solved in sql repo --- we no longer create symlinks or move files.

**Comment by jeromekelleher** [comment]
Closed by #1166.
